core/provisioning-java/src/main/java/org/apache/syncope/core/provisioning/java/data/GroupDataBinderImpl.java

```diff
@@ -460,12 +460,18 @@ public GroupTO getGroupTO(final String key) {
         return getGroupTO(groupDAO.authFind(key), true);
     }
 
-    protected static void populateTransitiveResources(
-            final Group group, final Any<?> any, final Map<String, PropagationByResource<String>> result) {
+    protected void populateTransitiveResources(
```

Comment:
any reason to remove `static` here?

Reply:
Why is it needed static?

Reply:
Because I thought that there was no need to have it static, but now it's clear, going to keep it.